src/main/java/com/example/work9/service/UserService.java

```diff
@@ -11,5 +11,5 @@ public interface UserService {
 
     User createUser(String getName);
 
-    void update(int id, String name) throws Exception;
+    User updateUser(int id, String getName);
```

Comment:
nameのままでいい気がしましたがどうでしょうか？
```suggestion
    User updateUser(int id, String name);
```

Reply:
修正しました！ただ理由を聞かれるとうまく言語化できないレベルです・・・😰。create処理の時はnameを渡すのにgetterを使いましたが今回はそのままnameで良い・・？？この辺はもっと慣れが必要なのでしょうか？😭

Reply:
updateUserを呼び出すときにgetterを使うのはOKです！

ただ、メソッド名として定義するときは変数の名前を決めればよいので
```java
User updateUser(int id, String name);
```
のようにするのがいいです。

> この辺はもっと慣れが必要なのでしょうか？😭

慣れ、だと思います 😭 
私が新卒当時、大変お世話になった先輩いわく、たくさんコードを読みなさいと言われました。
今でもその言葉は大事にしています。

ryoさんにはたくさんコードを読むために、開発ができる現場に早く入ってほしいものです 🚀

Reply:
ご確認ありがとうございます！そのご説明で理解しました。と同時に自分の認識違いも気付けました。
先のコメントの意味もわかりました！
早く現場に入れるように頑張ります！💪